### ISSUE TITLE
Needs requests as dependency?

### ISSUE THREAD

**Issue by endolith** [FabriceSalvaire/PySpice#266]
### Environment (OS, Python version, PySpice version, simulator)

Windows 10 64-bit, conda 4.9.2, python 3.9.1

### Expected Behaviour

`pyspice-post-installation --install-ngspice-dll` should install the DLL

### Actual Behaviour

It complained that `requests` was not installed

```
(pyspice) λ pyspice-post-installation --install-ngspice-dll
Traceback (most recent call last):
  File "C:\Anaconda3\envs\pyspice\Scripts\pyspice-post-installation-script.py", line 33, in <module>
    sys.exit(load_entry_point('PySpice==1.4.3', 'console_scripts', 'pyspice-post-installation')())
  File "C:\Anaconda3\envs\pyspice\Scripts\pyspice-post-installation-script.py", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "C:\Anaconda3\envs\pyspice\lib\importlib\metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "C:\Anaconda3\envs\pyspice\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "C:\Anaconda3\envs\pyspice\lib\site-packages\PySpice\Scripts\pyspice_post_installation.py", line 35, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

### Steps to reproduce the behaviour

I ran https://pyspice.fabrice-salvaire.fr/releases/v1.4/installation.html#on-windows

I created a blank conda environment, activated it, ran `conda install -c conda-forge pyspice`, then ran `pyspice-post-installation --install-ngspice-dll`

**Comment by umesh-timalsina** [comment]
I can attest to same for Linux.

**Comment by fdion** [comment]
Same issue on linux using conda, post install it errors out doing `pyspice-post-installation --check-install`
```
ModuleNotFoundError: No module named 'requests'
```

Workaround is to `conda install requests`

**Comment by FabriceSalvaire** [comment]
Fixed in PR #270
